src/main/webapp/script.js

```diff
@@ -0,0 +1,30 @@
+function getOriginalRecipe() {
+  const key = document.getElementById("key").value;
+  if (key !== "") {
```

Comment:
fix according to https://stackoverflow.com/questions/154059/how-can-i-check-for-an-empty-undefined-null-string-in-javascript